development.ini

```diff
@@ -15,8 +15,15 @@ pyramid.includes =
     pyramid_debugtoolbar
     pyramid_celery
 
-git.path = %(here)s/repo/
-git.content_repo_url = https://github.com/universalcore/unicore-cms-content-ffl-tanzania
+# git.path can be a local path or a URL that points to a unicore.distribute repo endpoint.
+# The index on Elasticsearch corresponds to the basename (minus .git or .json suffix).
+# For that reason, the basename must be alphanumeric and lowercase. It may also not
+# start with _.
+git.path = %(here)s/repo
```

Comment:
You're right. It was used by `init_repository` but that has been deleted. Want me to remove it? As long as we keep `git.path` we'll have backwards compatibility.